src/main/scala/com/johnsnowlabs/reader/SparkNLPReader.scala

```diff
@@ -70,40 +81,64 @@ class SparkNLPReader(params: java.util.Map[String, String] = new java.util.HashM
     *   Parameter with custom configuration
     */
 
+  private var outputColumn = "reader"
+
+  def setOutputColumn(value: String): Unit = {
+    require(value.nonEmpty, "Result column name cannot be empty.")
+    outputColumn = value
+  }
+
+  def getOutputColumn: String = outputColumn
```

Comment:
As far as I understand, the behavior of this class now, is that the output column is determined by the type of reader that is used ("html", "email" etc).

If a user wants to define a column name for the reader regardless of type, will it ever be used?

Reply:
@DevinTDHa This is a bit tricky to understand. It was added to integrate `PartitionTransformer` with a regular Spark NLP pipeline using DocumentAssembler. Essentially, the output of `Partition` becomes the `input` of `PartitionTransformer`, allowing the pipeline to proceed to the next stage.